# ![sweetpotatovirome](http://bioinfo.bti.cornell.edu/static/img/logo_new.jpg)
[![GitHub R package version](https://img.shields.io/github/r-package/v/ricardoi/sweetpotato_virome?label=%3A%3A&logo=R&logoColor=blue&style=plastic)](https://www.r-project.org/)
#[![run with R](https://img.shields.io/github/r-package/v/ricardoi/sweetpotato_virome/edit/main/README.md?label=-&logo=R&logoColor=blue)]#(https://www.r-project.org/)
[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)


# Sub-Saharan Africa Sweetpotato Virome (SSA-SPV)

Pan-African sweetpotato virome: a network analysis approach for the analysis of communities of viruses

------------------------------------------------------------------------------------------------------

The Pan-African sweetpotato virome:

A project with Dr. Jan F. Kreuze from International Potato Center ([CIP](https://cipotato.org/)), Lima, \
and Dr. Karen A. Garrett and Ricardo I. Alcala from the department of Plant Pathology ([PLP](https://plantpath.ifas.ufl.edu/)), \
Emerging Pathogens Institute ([EPI](https://epi.ufl.edu/)) and the Food Systems Institute ([FSI](https://foodsystems.ifas.ufl.edu/)) at the [University of Florida](http://www.ufl.edu/).

The Pan-African sweetpotato virome [website](http://bioinfo.bti.cornell.edu/virome/index) serve as the repository of the Sub-Saharan\
Africa sweetpotato virome network analysis ([manuscript in prep.](http://www.pending.org)).

Summary:

A collection of 1300 viromes of swetpotato samples collected from eleven African countries were sequenced with Illumina short reads derived from small viral RNAs. The sequences were assembled with VirusDetect followed by secondary iterations allowing the classification of viruses according to the International Committee of Taxonomy of Viruses ([ICTV](https://talk.ictvonline.org/)).
We developed a network framework for the classification of virus sequences using the maximum pairwise similarity between the viral  contigs and the Reference Sequences with similarities above the species demarcation criteria. \
The identities of each viral contig was assigned taxonomically according to the closest reference sequences. These virus species clusters were quantified using the small RNA reads using RPKM quantification ().
We identified the plant virome of each sweetpotato sample, and all infected plants were analyzed it as a bipartite networks; one set of nodes indicating the host and another subset of nodes indicating the virus - the virus species complex - where a disconnected graph represents a mixed infection, otherwise a single infection.
